### PR TITLE
Update direct-routing-plan-media-bypass.md

### PR DESCRIPTION
**Pull request by NMuravlyannikov**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan-media-bypass.md [hunk 1]
```diff
@@ -151,10 +151,19 @@ Teams Transport Relays are always in the media path in the following scenarios:
 Ensure your SBC has access to the Transport Relays as described below.    
 
 
-## SIP Signaling: FQDNs and firewall ports
+## SIP Signaling: FQDNs
 
 For SIP signaling, the FQDN and firewall requirements are the same as for non-bypassed cases. 
 
+Direct Routing is offered in the following Office 365 envirionments:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
Direct Routing is offered in the following Office 365 environments:
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 2]
```diff
@@ -151,10 +151,19 @@ Teams Transport Relays are always in the media path in the following scenarios:
...
+- Office 365;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
- Office 365
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 3]
```diff
@@ -151,10 +151,19 @@ Teams Transport Relays are always in the media path in the following scenarios:
...
+- Office 365 GCC;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
- Office 365 GCC
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 4]
```diff
@@ -151,10 +151,19 @@ Teams Transport Relays are always in the media path in the following scenarios:
...
+- Office 365 GCC High;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
- Office 365 GCC High
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 5]
```diff
@@ -151,10 +151,19 @@ Teams Transport Relays are always in the media path in the following scenarios:
...
+- Office 365 DoD
+More about GCC, GCC High and DoD envirionments [on this link] (https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/office-365-us-government)
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
Read more about GCC, GCC High, and DoD environments [on this link] (https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/office-365-us-government).
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 6]
```diff
@@ -151,10 +151,19 @@ Teams Transport Relays are always in the media path in the following scenarios:
...
+
+### Office 365 and Office 365 GCC envirionments
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
### Office 365 and Office 365 GCC environments
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 7]
```diff
@@ -178,7 +187,43 @@ The FQDNs **sip.pstnhub.microsoft.com**, **sip2.pstnhub.microsoft.com**, and **s
 - 52.114.7.24
 - 52.114.14.70
 
-You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling. If your firewall supports DNS names, the FQDN **sip-all.pstnhub.microsoft.com** resolves to all the IP addresses above. You must use the following ports:
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling. If your firewall supports DNS names, the FQDN **sip-all.pstnhub.microsoft.com** resolves to all the IP addresses above.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
You need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling. If your firewall supports DNS names, the FQDN **sip-all.pstnhub.microsoft.com** resolves to all these IP addresses. 
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 8]
```diff
@@ -178,7 +187,43 @@ The FQDNs **sip.pstnhub.microsoft.com**, **sip2.pstnhub.microsoft.com**, and **s
 - 52.114.7.24
 - 52.114.14.70
 
-You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling. If your firewall supports DNS names, the FQDN **sip-all.pstnhub.microsoft.com** resolves to all the IP addresses above. You must use the following ports:
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling. If your firewall supports DNS names, the FQDN **sip-all.pstnhub.microsoft.com** resolves to all the IP addresses above. 
+
+### Office 365 GCC DoD envirionment
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
### Office 365 GCC DoD environment
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 9]
```diff
@@ -178,7 +187,43 @@ The FQDNs **sip.pstnhub.microsoft.com**, **sip2.pstnhub.microsoft.com**, and **s
...
+
+The connection point for Direct Routing is the following FQDN:
+
+**sip.pstnhub.dod.teams.microsoft.us** – Global FQDN. Note as the Office 365 DoD envirionment exists only in the US datacenters, there is no secondary and tertiary FQDNs
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
**sip.pstnhub.dod.teams.microsoft.us** – Global FQDN. As the Office 365 DoD environment exists only in the US data centers, there is no secondary and tertiary FQDNs.
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 10]
```diff
@@ -178,7 +187,43 @@ The FQDNs **sip.pstnhub.microsoft.com**, **sip2.pstnhub.microsoft.com**, and **s
...
+
+The FQDNs – sip.pstnhub.dod.teams.microsoft.us will be resolved to one of the following IP addresses:
+
+- 52.127.64.33
+- 52.127.68.34
+
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.dod.teams.microsoft.us resolves to all the IP addresses above.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
You need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.dod.teams.microsoft.us resolves to all these IP addresses. 
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 11]
```diff
@@ -178,7 +187,43 @@ The FQDNs **sip.pstnhub.microsoft.com**, **sip2.pstnhub.microsoft.com**, and **s
...
+**sip.pstnhub.dod.teams.microsoft.us** – Global FQDN. Note as the Office 365 DoD envirionment exists only in the US datacenters, there is no secondary and tertiary FQDNs
+
+The FQDNs – sip.pstnhub.dod.teams.microsoft.us will be resolved to one of the following IP addresses:
+
+- 52.127.64.33
+- 52.127.68.34
+
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.dod.teams.microsoft.us resolves to all the IP addresses above. 
+
+### Office 365 GCC High envirionment
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
### Office 365 GCC High environment
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 12]
```diff
@@ -178,7 +187,43 @@ The FQDNs **sip.pstnhub.microsoft.com**, **sip2.pstnhub.microsoft.com**, and **s
...
+
+The connection point for Direct Routing is the following FQDN:
+
+**sip.pstnhub.gov.teams.microsoft.us** – Global FQDN. Note as the GCC High envirionment exists only in the US datacenters, there is no secondary and tertiary FQDNs
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
**sip.pstnhub.gov.teams.microsoft.us** – Global FQDN. As the GCC High environment exists only in the US data centers, there is no secondary and tertiary FQDNs.
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 13]
```diff
@@ -178,7 +187,43 @@ The FQDNs **sip.pstnhub.microsoft.com**, **sip2.pstnhub.microsoft.com**, and **s
...
+
+The FQDNs – sip.pstnhub.gov.teams.microsoft.us will be resolved to one of the following IP addresses:
+
+- 52.127.88.59
+- 52.127.92.64
+
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.gov.teams.microsoft.us resolves to all the IP addresses above.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
You need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.gov.teams.microsoft.us resolves to all these IP addresses. 
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 14]
```diff
@@ -178,7 +187,43 @@ The FQDNs **sip.pstnhub.microsoft.com**, **sip2.pstnhub.microsoft.com**, and **s
...
+The FQDNs – sip.pstnhub.gov.teams.microsoft.us will be resolved to one of the following IP addresses:
+
+- 52.127.88.59
+- 52.127.92.64
+
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.gov.teams.microsoft.us resolves to all the IP addresses above. 
+
+## SIP Signaling: Ports
+
+Prt requiremtns are the same for all Office 365 envirionments, where Direct Routing is offered:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
Port requirements are the same for all Office 365 environments where Direct Routing is offered:
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 15]
```diff
@@ -178,7 +187,43 @@ The FQDNs **sip.pstnhub.microsoft.com**, **sip2.pstnhub.microsoft.com**, and **s
...
+- Office 365;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
- Office 365
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 16]
```diff
@@ -178,7 +187,43 @@ The FQDNs **sip.pstnhub.microsoft.com**, **sip2.pstnhub.microsoft.com**, and **s
...
+- Office 365 GCC;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
- Office 365 GCC
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 17]
```diff
@@ -178,7 +187,43 @@ The FQDNs **sip.pstnhub.microsoft.com**, **sip2.pstnhub.microsoft.com**, and **s
...
+- Office 365 GCC High;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
- Office 365 GCC High
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 18]
```diff
@@ -206,9 +251,22 @@ Note: If you have a network device that translates the client's source ports, pl
 
 ### Requirements for using Transport Relays
 
-Transport Relays are in the same range as Media Processors (for non-bypass cases):  52.112.0.0 /14 (IP addresses from 52.112.0.1 to 52.115.255.254).
+Transport Relays are in the same range as Media Processors (for non-bypass cases): 
+
+### Office 365 and Office 365 GCC envirionments
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
### Office 365 and Office 365 GCC environments
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 19]
```diff
@@ -206,9 +251,22 @@ Note: If you have a network device that translates the client's source ports, pl
...
+
+-52.112.0.0 /14 (IP addresses from 52.112.0.1 to 52.115.255.254)
+
+## Office 365 GCC DoD envirionment
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
## Office 365 GCC DoD environment
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 20]
```diff
@@ -206,9 +251,22 @@ Note: If you have a network device that translates the client's source ports, pl
...
+
+- 52.127.64.0/21
+
+### Office 365 GCC High envirionment
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
### Office 365 GCC High environment
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 21]
```diff
@@ -232,9 +290,21 @@ You need to open ports 3478 and 3479 for transitioning. When Microsoft introduce
 Media Processors are always in the media path for voice applications and for Web cleints (for exampe, Teams cleint in Edge or Google Chrome). The requirements are the same as for non-bypass configuration.
 
 
-The IP range for media traffic is 52.112.0.0 /14 (IP addresses from 52.112.0.1 to 52.115.255.254).
+The IP range for media traffic is 
+
+### Office 365 and Office 365 GCC envirionments
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
### Office 365 and Office 365 GCC environments
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 22]
```diff
@@ -232,9 +290,21 @@ You need to open ports 3478 and 3479 for transitioning. When Microsoft introduce
...
+
+-52.112.0.0 /14 (IP addresses from 52.112.0.1 to 52.115.255.254)
+
+## Office 365 GCC DoD envirionment
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
## Office 365 GCC DoD environment
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 23]
```diff
@@ -232,9 +290,21 @@ You need to open ports 3478 and 3479 for transitioning. When Microsoft introduce
...
+
+- 52.127.64.0/21
+
+### Office 365 GCC High envirionment
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
### Office 365 GCC High environment
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 24]
```diff
@@ -232,9 +290,21 @@ You need to open ports 3478 and 3479 for transitioning. When Microsoft introduce
...
+
+- 52.127.88.0/21
 
-The port range of the Media Processors is shown in the following table:
+The port range of the Media Processors (applicable to all envirionments) is shown in the following table:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
The port range of the Media Processors (applicable to all environments) is shown in the following table:
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 25]
```diff
@@ -206,9 +251,22 @@ Note: If you have a network device that translates the client's source ports, pl
...
+
+- 52.127.64.0/21
+
+### Office 365 GCC High envirionment
+
+- 52.127.88.0/21
 
-The port range of the Teams Transport Relays is shown in the following table:
+
+The port range of the Teams Transport Relays (applicable to all envirionments) is shown in the following table:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
The port range of the Teams Transport Relays (applicable to all environments) is shown in the following table:
```

File: Teams/direct-routing-plan-media-bypass.md (hunk 5 above)
Comment (nenonix) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
Learn more about [Office 365 and US Government environments](https://docs.microsoft.com/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/office-365-us-government) such as GCC, GCC High, and DoD.

**Pull request merged**